pkg/common/container.go

```diff
@@ -41,21 +49,36 @@ func getOSPVersion(parsedVersion string) (ospdirectorv1beta1.OSPVersion, error)
 }
 
 // GetVersionFromImageURL -
-func GetVersionFromImageURL(imageURL string) (ospdirectorv1beta1.OSPVersion, error) {
+func GetVersionFromImageURL(r ReconcilerCommon, imageURL string) (ospdirectorv1beta1.OSPVersion, error) {
```

Comment:
Not important, but a suggestion: maybe call it `GetOSPVersionFromImageURL` just to be more specific on what the version refers to?

Reply:
makes sense, done